Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -634,7 +674,7 @@ public void cleanWebAppsOnLinux() {
      * An issue is logged at https://github.com/Azure/azure-sdk-for-java/issues/1755 .
      * Remove all tags here to make it work.
      */
-    private void clearTags(@NotNull final WebApp app) {
+    private void clearTags(@NotNull final WebAppBase app) {
```

Comment:
So looks like we don't need this now.

Reply:
Will update the issue link to https://github.com/Azure/azure-libraries-for-java/issues/508